src/main/java/org/myrobotlab/service/WebGui.java

```diff
@@ -266,15 +266,15 @@ static public ServiceType getMetaData() {
 
   transient Map<String, Map<String, Panel>> desktops;
 
-  transient ApiFactory api = null;
+  transient ApiFactory apix = null;

```

Comment:
If this is no longer used, the reference here should be removed.

Reply:
aye .. already removed locally - will be in next pr